data/download_data.sh

```diff
@@ -1,13 +1,14 @@
-dataset = $1
+dataset=$1
 
-LINK='https://zenodo.org/record/6903423/files/${1}?download=1'
+LINK="https://zenodo.org/record/6903423/files/${dataset}"
 
-if [[ $LINK == *.pkl ]]; then
+if [[ $dataset == *.pkl ]]; then
     mkdir datasets
     wget $LINK -O datasets/${dataset}
 else
-    if [[ $LINK == *.tar.gz ]]; then
+    if [[ $dataset == *.tar.gz ]]; then
+        echo "downloading ${dataset}..."
         wget $LINK
-        tar -xzvf $LINK
+        tar -xzvf $dataset
```

Comment:
the name gets downloaded as just the dataset name instead of the full link so this is the path we need to unzip